src/frontends/tensorflow/src/openvino_conversions.hpp

```diff
@@ -43,9 +43,9 @@ void convert_nchw_to_hw(const std::vector<T>& src, std::vector<size_t>& dst) {
 }
 }  // namespace detail
 
-void convert_nhwc_to_nchw(const std::string& op_name, bool need_convert, ov::Output<ov::Node>& ng_input);
+void convert_nhwc_to_nchw(bool need_convert, ov::Output<ov::Node>& input_);
```

Comment:
nit: ng_input was a better name why change?

Reply:
there is no longer NGraph term and it looks ugly, imho:)

Reply:
Why line 48 isn't changed then? Ending with underscore is pretty ugly too, imho :)

Reply:
Agree, I will do it:)

Reply:
Fixed. Thanks.